src/index.stories.js

```diff
@@ -1,6 +1,50 @@
 import React from 'react';
 import { storiesOf } from '@kadira/react-native-storybook';
+// import { Router, Route, Actions, routerReducer } from 'react-native-redux-router';
+import { Scene, Router, Actions } from 'react-native-router-flux';
+import { Button, View } from 'react-native';
+import { createStore, combineReducers } from 'redux';
+import { Provider } from 'react-redux';
 
+import reducers from './reducers';
+import ExerciseApp from './containers/ExerciseApp';
+import Splash from './components/exercises/Splash';
 import App from '.';
 
+// default
 storiesOf('App', module).add('default', () => <App />);
+
+// router demonstration
+const reducer = combineReducers(reducers);
+const store = createStore(reducer);
```

Comment:
I've ended up making a completely different story for router, which is much simpler and better represents the router functionality (and doesn't use store at all)
But yes, I agree - initialisation should've been in the story itself.